server/src/internalClusterTest/java/org/elasticsearch/cluster/routing/allocation/decider/WriteLoadConstraintDeciderIT.java

```diff
@@ -663,12 +696,29 @@ private NodeUsageStatsForThreadPools createNodeUsageStatsForThreadPools(
      */
```

Comment:
Method comment needs update

Reply:
Fixed in 29a79e13f14e60d6856ff411a61405c7ba317aaa